apollo-router/src/plugins/telemetry/tracing/datadog_exporter/exporter/model/unified_tags.rs

```diff
@@ -83,41 +83,3 @@ impl UnifiedTagEnum {
         }
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
These were effectively tests for `std::env::set_var()` and `std::env::var()`. If they were removed or mocked out, there would be so little business logic left that it's not worth IMO trying to have unit tests for it.